### PR TITLE
MM-12050: Add tooltips to post info overlay buttons.

### PR DESCRIPTION
**Pull request by grundleborg**
#### Summary
Add tooltips to post info overlay buttons.

#### Ticket Link
https://mattermost.atlassian.net/browse/MM-12050

#### Checklist
- [x] Ran `make check-style` to check for style errors (required for all pull requests)
- [x] Ran `make test` to ensure unit and component tests passed
- [x] Added or updated unit tests (required for all new features)
- [x] Has UI changes
- [x] Includes text changes and localization file updates

### CODE REVIEW

File: components/post_view/post_info/post_info.jsx [hunk 1]
```diff
@@ -224,6 +224,7 @@ export default class PostInfo extends React.PureComponent {
                             <button
                                 className='reacticon__container color--link style--none'
                                 onClick={this.toggleEmojiPicker}
+                                title={Utils.localizeMessage('post_info.tooltip.add_reactions', 'Add Reactions')}
```
Comment (jespino) [inline components/post_view/post_info/post_info.jsx]: Maybe "Add reactions" must be singular instead of plural.

File: i18n/en.json [hunk 2]
```diff
@@ -2816,6 +2816,9 @@
   "post_info.reply": "Reply",
   "post_info.system": "System",
   "post_info.unpin": "Un-pin from channel",
+  "post_info.tooltip.add_reactions": "Add Reactions",
```
Comment (jespino) [inline i18n/en.json]: Not important because it will be automatically solved soon, but It should be in alphabetical order inside the file.

Comment (grundleborg) [reply i18n/en.json]: Yeah I just let it do that automatically.

File: components/post_view/post_info/post_info.jsx (hunk 1 above)
Comment (grundleborg) [reply components/post_view/post_info/post_info.jsx]: The ticket specified "Add Reactions" plural, but I agree you might have a point. @esethna which one should it be?

Comment (esethna) [reply components/post_view/post_info/post_info.jsx]: Thanks, good catch. Lets go with "reaction", singular

**Comment by grundleborg** [comment]
@esethna fixed.

**Comment by esethna** [comment]
Looks like a build error? ^

**Pull request merged**